### PR TITLE
feat(seo): improve notion render urls for seo boost

### PR DESCRIPTION
**Pull request by BleedingDev**
weird props getter is for getting alt of the image nested in link

### CODE REVIEW

File: pages/glossary/[id].tsx [hunk 1]
```diff
@@ -39,10 +40,10 @@ export async function getStaticPaths() {
...
   })
 
-  const paths = Object.keys(pages)
-    .map((pageId) => mapPageUrl(pageId))
-    .filter((path) => path && path !== "/")
-    .map((path) => `/glossary${path}`)
+  const paths = Object.entries(pages)
+    .map(([key, page]) => (page && key ? [mapPageUrl(key), getPageTitle(page)] : ["", ""]))
+    .filter(([id, title]) => id && title && title !== "Glossary")
+    .map(([id, path]) => `/glossary/${path.toLowerCase().replaceAll(" ", "-")}-${id.replaceAll("/", "")}`)
```
Comment (devodii) [inline pages/glossary/[id].tsx:46]: This is looking too complex.

File: pnpm-lock.yaml [hunk 2]
```diff
@@ -151,6 +151,9 @@ devDependencies:
   '@ianvs/prettier-plugin-sort-imports':
     specifier: ^4.1.1
     version: 4.1.1(prettier@3.1.0)
+  '@total-typescript/ts-reset':
+    specifier: ^0.5.1
+    version: 0.5.1
```
Comment (devodii) [inline pnpm-lock.yaml:156]: What do we need this for?

**Review by devodii — CHANGES_REQUESTED** [review]

File: pnpm-lock.yaml (hunk 2 above)
Comment (BleedingDev) [reply pnpm-lock.yaml:156]: Better Typescript defaults.

File: pages/glossary/[id].tsx (hunk 1 above)
Comment (BleedingDev) [reply pages/glossary/[id].tsx:46]: I might extract logic into some util function to share it.

**Pull request merged**